pkgs/applications/graphics/gimp/wrapper.nix

```diff
@@ -1,7 +1,7 @@
 { stdenv, lib, symlinkJoin, gimp, makeWrapper, gimpPlugins, gnome3, plugins ? null}:
 
 let
-allPlugins = lib.filter (pkg: builtins.isAttrs pkg && pkg.type == "derivation" && !pkg.meta.broken or false) (lib.attrValues gimpPlugins);
+allPlugins = lib.filter (pkg: lib.isDerivation pkg && !pkg.meta.broken or false) (lib.attrValues gimpPlugins);
```

Comment:
This change is necessary because `gimpPlugins` was `makeOverride`'d, meaning it has a `.override`, which previously was a function, but now is a functor attrset (but without a `.type`), which this function wasn't able to handle before. The `lib.isDerivation` function properly checks for that.